etc/config-template.toml

```diff
@@ -108,10 +108,10 @@
 # A threshold to gc stale raft log, must >= 1.
 # raft-log-gc-threshold = 50
 # When entry count exceed this value, gc will be forced trigger.
-# raft-log-gc-count-limit = 196608
+# raft-log-gc-count-limit = 48000
 # When the approximate size of raft log entries exceed this value, gc will be forced trigger.
 # It's recommanded to set it to 3/4 of region-split-size.
-# raft-log-gc-size-limit = "192MB"
+# raft-log-gc-size-limit = "48MB"
```

Comment:
I think the default value is 72MB.

Reply:
Follow https://github.com/pingcap/tikv/pull/2017/files#diff-d4c79fcc9faffa48ae170c90d8c0683aL96

Reply:
But it's conflict with https://github.com/pingcap/tikv/pull/2313/files#diff-c9a441bf6785be28f64f5b07d40725a2R118